public/app/features/dashboard/components/ShareModal/ShareExport.tsx

```diff
@@ -51,13 +60,17 @@ export const ShareExport = memo(({ dashboard, panel, onDismiss }: Props) => {
     }
   };
 
-  const openSaveAsDialog = (dash: any) => {
+  const openSaveAsDialog = (dash: unknown) => {
     const dashboardJsonPretty = JSON.stringify(dash, null, 2);
     const blob = new Blob([dashboardJsonPretty], {
       type: 'application/json;charset=utf-8',
     });
     const time = new Date().getTime();
-    saveAs(blob, `${dash.title}-${time}.json`);
+    const title =
+      typeof dash === 'object' && dash !== null && 'title' in dash && typeof dash.title === 'string'
```

Comment:
This could be extracted into a standalone type guard: 
```ts

 interface WithTitle {
    title: string;
  }

function hasTitle(obj: unknown): obj is WithTitle {
    return typeof obj === 'object' && obj !== null && 'title' in obj && typeof obj.title === 'string';
  }
```

Altho this is not relevant to the existing PR so maybe better be left out.